utils/augmentation.py

```diff
@@ -60,10 +64,28 @@ def __call__(self, sample):
             return sample
 
 
+class Normalize(object):
+    """Normalize Image with Mean and STD and similar to Pytorch(transform.Normalize) function """
+    def __init__(self, mean, std):
+        self.mean = mean
+        self.std = std
+
+    def __call__(self, sample):
+        if self.mean or self.std != []:
+            sat_img = (sample['sat_img'] - self.mean) / self.std
+            map_img = sample['map_img']
+            return {'sat_img': sat_img, 'map_img': map_img}
+        else:
+            return sample
+
+
 class ToTensorTarget(object):
     """Convert ndarrays in sample to Tensors."""
+    def __init__(self, num_classes):
+        self.num_classes = num_classes
 
     def __call__(self, sample):
         sat_img = np.float32(np.transpose(sample['sat_img'], (2, 0, 1)))
         map_img = np.int64(sample['map_img'])
+        map_img[map_img > self.num_classes] = 0
```

Comment:
Not sure I understand the purpose of this line. What if map_img carries ignore_index values that are bigger than num_classes (e.g. 255, which is the case in our current dataset)

Reply:
it was hacky fix, that solved the problem with using other loss functions such as cross_entropy loss_fn